sources/functions/short

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#please make sure to give this full paths
+function rm_if_exists () {
+    path="$1"
+    if [[ -e $path ]]; then
+      rm -rf $path
```

Comment:
I think it might be sensible to quote the `$path` variable here in line 6 and 7, especially with a recursive `rm` in line 7.

The expansion of any path with spaces would fail. The use may ever present the problem but i think it would be good practice to quote a path expansion.

Reply:
Nice catch, that was my intention but it slipped me. Thanks!